shiksha-website/shiksha-frontend/src/app/shared/components/baseline-survey/baseline-survey.component.html

```
@@ -1,294 +1,266 @@
+        <!-- Q1: plans -->
+        <ng-container *ngIf="currentStep === 0">
+          <h2 class="question">How do you currently prepare your lesson plans?</h2>
+          <p class="hint">Select all that apply</p>
+          <div class="options">
+            <div class="opt check" *ngFor="let o of planOptions" [class.selected]="isChecked('plans', o)"
+              (click)="toggleArray('plans', o)">
+              <span class="mark"></span><span>{{ o }}</span>
+            </div>
+          </div>
+          <div class="other-box" *ngIf="showPlansOther">
+            <input type="text" formControlName="plansOther" placeholder="Please specify" />
```

HTMLHint (1.9.2)

[warning] No matching [ label ] tag found.

(input-requires-label)